### PR TITLE
These channels helped me to level up my Front End Skills

### PR DESCRIPTION
**Pull request by sojinsamuel**
These channels explain js, angular, react, and typescript in a solid way + it's presented in an easy-to-digest format for absolute beginners who are starting out their coding journey

### CODE REVIEW

**Review by KendallDoesCoding — COMMENTED** [review]
The description scheme is not like the above ones and needs a little shortning which I will help you with in a bit. Other then that, looks good

File: README.md [hunk 1]
```diff
@@ -40,3 +40,8 @@ You can add a YouTuber that helped you begin coding in the below list by followi
 | [JS Bangladesh](https://www.youtube.com/c/JSBangladesh) | Makes content around **Web Development** but focuses mostly on **Javascript** | Bengali |
 | [Anisul Islam](https://www.youtube.com/c/anisulislamrubel) | Mostly makes content around **Web Development** but also makes discussion videos & content around other programming languages from time to time | Bengali |
 | [Javascript Mastery](https://www.youtube.com/c/JavaScriptMastery) | Produces great content related to **Javascript Web Development** and **Application Development** | English |
+| [Kent C. Dodds](https://www.youtube.com/user/kentdoddsfamily/videos) | In this YouTube channel, you’ll find live streams and videos on beginner and advanced topics ranging from **JavaScript** to **React**, **TypeScript**, and **dev tips** | English |
```
Comment (KendallDoesCoding) [inline README.md:43]:
```suggestion
| [Kent C. Dodds](https://www.youtube.com/user/kentdoddsfamily/videos) | Makes content around **JavaScript**, **React**, **TypeScript**, and **dev tips** | English |
```

File: README.md [hunk 2]
```diff
@@ -40,3 +40,8 @@ You can add a YouTuber that helped you begin coding in the below list by followi
...
+| [Double South](https://www.youtube.com/channel/UCzroHLKLlfA-hR7S4if6qww) | In this channel, Itsey teaches **beginner friendly** JavaScript, HTML, and CSS | Pidgin English |
```
Comment (KendallDoesCoding) [inline README.md:44]:
```suggestion
| [Double South](https://www.youtube.com/channel/UCzroHLKLlfA-hR7S4if6qww) | Makes content around **beginner friendly** JavaScript, HTML, and CSS | Pidgin English |
```

File: README.md [hunk 3]
```diff
@@ -40,3 +40,8 @@ You can add a YouTuber that helped you begin coding in the below list by followi
...
+| [The New Boston](https://www.youtube.com/user/thenewboston) | This channel provides tutorials in abundance for the following sub-topics – **ReactJS**, **ES6** (ECMA Script), **AngularJS**, Working with **NodeJS**, **Vanilla JavaScript** and **Docker**| English |
```
Comment (KendallDoesCoding) [inline README.md:45]:
```suggestion
| [The New Boston](https://www.youtube.com/user/thenewboston) | Makes content around **ReactJS**, **ES6** (ECMA Script), **AngularJS**, Working with **NodeJS**, **Vanilla JavaScript** and **Docker**| English |
```

File: README.md [hunk 4]
```diff
@@ -40,3 +40,8 @@ You can add a YouTuber that helped you begin coding in the below list by followi
...
+| [LearnCode.academy](https://www.youtube.com/c/learncodeacademy/videos) | This channel provides tutorials on everything you need to kick start your **web development** journey, but not limited to JavaScript | English |
```
Comment (KendallDoesCoding) [inline README.md:46]:
```suggestion
| [LearnCode.academy](https://www.youtube.com/c/learncodeacademy/videos) |Makes content around **web development**.| English |
```

File: README.md [hunk 5]
```diff
@@ -40,3 +40,8 @@ You can add a YouTuber that helped you begin coding in the below list by followi
...
+| [Amigoscode](https://www.youtube.com/c/amigoscode) | In the past few years, nelson has programmed in various programming languages such as PHP, Python, C# and primarily writes Java and JavaScript, focusing on Distributed Systems hosted on the cloud. | English |
```
Comment (KendallDoesCoding) [inline README.md:47]:
```suggestion
| [Amigoscode](https://www.youtube.com/c/amigoscode) | In the past few years, Nelson has programmed in various programming languages such as PHP, Python, C# and primarily writes Java and JavaScript, focusing on Distributed Systems hosted on the cloud. | English |
```

Comment (KendallDoesCoding) [inline README.md:47]: I don't know what edits we can make to this line to make it shorter, any ideas @AnimeshKumar923 https://github.com/orgs/collab-community/teams/coding-youtubers?

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (AnimeshKumar923) [reply README.md:47]: > I don't know what edits we can make to this line to make it shorter, any ideas @AnimeshKumar923 https://github.com/orgs/collab-community/teams/coding-youtubers?


You can assign it to me. I'll look into it after sometimea nd fix everything.

A bit busy with college stuff...😅

Comment (KendallDoesCoding) [reply README.md:47]: Ok cool, I think my changes are good for the other ones, but struggling for edits with line 47 lol

Comment (AnimeshKumar923) [reply README.md]: > Ok cool, I think my changes are good for the other ones, but struggling for edits with line 47 lol

Don't merge this pr, keep it open.

I have some new ideas which I'll be sharing tonight and will be of great help.

Just let me finish this college work...🗿

Comment (KendallDoesCoding) [reply README.md]: > > Ok cool, I think my changes are good for the other ones, but struggling for edits with line 47 lol
> 
> Don't merge this pr, keep it open.
> 
> I have some new ideas which I'll be sharing tonight and will be of great help.
> 
> Just let me finish this college work...🗿

Yeah, I was anyway merge only once @SojinSamuel commit's my requested changes, I'll keep it open till your reply!

Comment (AnimeshKumar923) [reply README.md]: Thank you for your patience....

**Comment by adityajoshi-08** [comment]
Thanks for the contribution Samuel!

**Pull request merged**